content/08.transmission.md

```diff
@@ -284,3 +284,38 @@ COVID-19 can thus be considered a "syndemic", or a synergistic interaction betwe
 As a result, it is not surprising that people from minoritized backgrounds and/or with certain pre-existing conditions are more likely to suffer severe effects of COVID-19, but these "risk factors" are likely to be causally linked to poverty.
 
 #### Access to Treatment
+
+Finally, COVID-19 outcomes can be influenced by access to healthcare.
+Receiving care for COVID-19 can, but does not always, include receiving a positive test for the SARS-CoV-2 virus.
+For example, it is common to see treatment guidelines for suspected cases regardless of whether the presence of SARS-CoV-2 has been confirmed (e.g., [@doi:10.1016/j.ajem.2020.03.036]).
+Whether and where a patient is diagnosed can depend on their access to testing, which can vary both between and within countries.
+In the United States, it is not always clear whether an individual will have access to free testing [@url:https://www.npr.org/sections/health-shots/2020/06/19/880543755/insurers-may-only-pay-for-coronavirus-tests-when-theyre-medically-necessary; @url:https://www.commonwealthfund.org/blog/2020/private-health-insurance-coverage-covid-19-public-health-emergency].
+The concern has been raised that more economic privilege is likely to correspond to increased access to testing, at least within the United States [@doi:https://doi.org/10.1016/j.jaad.2020.04.046].
+This is supported by the fact that African Americans seem to be more likely to be diagnosed in the hospital, while other groups were more likely to have been diagnosed in ambulatory settings in the community [@doi:10.1377/hlthaff.2020.00598].
+Any delays in treatment are a cause for concern [@doi:10.1016/j.jaad.2020.04.046], which could potentially be increased by an inability to acquire testing because in the United States, insurance coverage for care received can depend on a positive test [@url:https://www.hrsa.gov/coviduninsuredclaim/frequently-asked-questions].
+
+Another important question is whether patients with moderate to severe cases are able to access hospital facilities and treatments, to the extent that they have been identified.
+Early findings from China as of February 2020 suggested the COVID-19 mortality rate to be much lower in the most developed regions of the country [@doi:10/ggqscd].
```

Comment:
If anyone is able to access the Chinese CDC report, it might have more info: <http://2019ncov.chinacdc.cn/2019-nCoV/> but I can't get it to load. The correspondence in the Lancet (cited) doesn't go into how they conducted testing.

```suggestion
Early findings from China as of February 2020 indicated that the COVID-19 mortality rate might be lower in the most developed regions of the country [@doi:10/ggqscd], although it's not clear whether this discrepancy could be influenced by rates of testing.
```

Reply:
I also couldn't get it to load! I think it's OK to just say something like "however reported mortality is generally an estimate of CFR, which is dependent on rates of testing," based on the Lancet report.

Reply:
![ChinesesCDC](https://user-images.githubusercontent.com/62627112/96650547-0d105000-1301-11eb-9511-db63b3baeded.png)

Reply:
I loaded it but it wasn't helpful as you can see :D

Reply:
Thank you so much for trying, @RLordan! I am definitely not sure how to interpret that, haha.

Thanks @SiminaB for the feedback on the text!
```suggestion
Early findings from China as of February 2020 suggested the COVID-19 mortality rate to be much lower in the most developed regions of the country [@doi:10/ggqscd], although reported mortality is generally an estimate of CFR, which is dependent on rates of testing.
```